pynest/examples/Potjans_2014/stimulus_params.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+#
+# stimulus_params.py
+#
+# This file is part of NEST.
+#
+# Copyright (C) 2004 The NEST Initiative
+#
+# NEST is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# NEST is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+
+'''
+microcicuit stimulus parameters
+-------------------------------
+
+Stimulus parameters for the microcircuit.
+
+Hendrik Rothe, Hannah Bos, Sacha van Albada; May 2016
+'''
+
+import numpy as np
+from network_params import net_dict
+
+stim_dict = {
+    # Turn thalamic input on or off (True or False).
+    'thalamic_input': False,
+    # Turn DC input on or off (True or False).
+    'dc_input': False,
+    # Number of thalamic neurons.
+    'n_thal': 902,
+    # Mean amplitude of the thalamic postsynaptic potential (in mV).
+    'PSP_th': 0.15,
+    # Standard deviation of the postsynaptic potential (in relative units).
+    'PSP_sd': 0.1,
+    # Start of the thalamic input (in ms).
+    'th_start': 700.0,
```

Comment:
I kept the thalamus stimulus interval, since this is the stimulus interval of the sli version. I therefore changed the interval of the spike raster plot.